planning/behavior_path_planner/autoware_behavior_path_goal_planner_module/src/goal_planner_module.cpp

```diff
@@ -808,17 +809,20 @@ void GoalPlannerModule::updateData()
         parameters_.th_stopped_velocity),
       std::move(lane_parking_response), std::move(freespace_parking_response));
   }
-  auto & ctx_data = context_data_.value();
+  const auto & ctx_data = context_data_.value();
+  goal_searcher.update(
+    goal_candidates_, occupancy_grid_map_, planner_data_, ctx_data.static_target_objects);
+  auto & ctx_data_mut = context_data_.value();
```

Comment:
memo for me:

this is moved from planPullOverAsOutput. calculating in updateData() is clearer

![image](https://github.com/user-attachments/assets/b27b5f65-a0c9-422a-84cf-81aa477a2410)